README.md

```diff
@@ -34,7 +73,7 @@ After making changes:
 
 1. Bump the version in `package.json`
 2. `vsce package` (select `y`; a `.vsix` file will be created) 
-3. `cmd+shift+p` > "Extensions: Install from VSIX" > select the `.vsix` file
+3. `cmd+shift+p` -> "Extensions: Install from VSIX" > select the `.vsix` file
 4. reload window
 
 <br />
```

Comment:
What do you think about outsourcing this `Local Development` section to another file, like `Development.md` or to the wiki? This way, people installing the extension wouldn't see this - in their eyes - unnecessary information.

Reply:
yes, would be nice